assets/sidecar/developer-hub-app-config-patches.yaml

```diff
@@ -0,0 +1,113 @@
+    auth:
+      environment: production
+      providers:
+        github:
+          production:
+            clientId: ${GITHUB__APP__CLIENT__ID}
+            clientSecret: ${GITHUB__APP__CLIENT__SECRET}
+    # start of delta from ai-rhdh-installer
+    backend:
+      auth:
+        externalAccess:
+          - type: static
+            options:
+              token: ${RHDH_TOKEN}
+              subject: admin-curl-access
+      reading:
+        allow:
+          - host: example.com
+          - host: '*.mozilla.org'
+          - host: '*.openshift.com'
+          - host: '*.openshiftapps.com'
+          - host: '10.128.*'
+          - host: '127.0.0.1:9090'
+          - host: '127.0.0.1:8080'
+          - host: '127.0.0.1:7070'
+          - host: 'localhost:9090'
+          - host: 'localhost:8080'
+          - host: 'localhost:7070'
+    # end of delta from ai-rhdh-installer
+    dangerouslyAllowSignInWithoutUserInCatalog: false
+    catalog:
+      rules:
+        - allow:
+            - Component
+            - System
+            - Group
+            - Resource
+            - Location
+            - Template
+            - API
+      locations:
+        - type: url
+          target: https://github.com/redhat-ai-dev/ai-lab-template/blob/main/all.yaml
+      providers:
+        github:
+          providerId:
+            organization: ${GITHUB__ORG__NAME}
+            schedule:
+              frequency:
+                minutes: 30
+              initialDelay:
+                seconds: 15
+              timeout:
+                minutes: 15
+        githubOrg:
+          githubUrl: https://${GITHUB__HOST}
+          orgs:
+            - ${GITHUB__ORG__NAME}
+          schedule:
+            frequency:
+              minutes: 30
+            initialDelay:
+              seconds: 15
+            timeout:
+              minutes: 15
+    proxy:
+      endpoints:
+        '/quay/api':
+          target: 'https://quay.io'
+          headers:
+            X-Requested-With: 'XMLHttpRequest'
+            Authorization: Bearer ${QUAY__API_TOKEN}
+          changeOrigin: true
+          # Change to "false" in case of using self hosted quay instance with a self-signed certificate
+          secure: true
+    quay:
+      # The UI url for Quay, used to generate the link to Quay
+      uiUrl: 'https://quay.io'
+    techdocs:
+      builder: 'local'
+      generator:
+        runIn: 'local'
+      publisher:
+        type: 'local'
+    integrations:
+      github:
+        - host: ${GITHUB__HOST}
+          apps:
+            - appId: ${GITHUB__APP__ID}
+              clientId: ${GITHUB__APP__CLIENT__ID}
+              clientSecret: ${GITHUB__APP__CLIENT__SECRET}
+              webhookUrl: ${GITHUB__APP__WEBHOOK__URL}
+              webhookSecret: ${GITHUB__APP__WEBHOOK__SECRET}
+              privateKey: ${GITHUB__APP__PRIVATE_KEY}
+    signInPage: github
+kind: ConfigMap
+metadata:
+  name: developer-hub-app-config
+  namespace: ai-rhdh
```

Comment:
nit: The namespace should be configurable

Reply:
yeah I'm assuming you'll make it configurable when you transfer it to your new stuff @thepetk 

I can just remove the setting of the namespace if you like